pages/clustering/replication/experimental-replication.mdx

```diff
@@ -1,15 +1,24 @@
 ---
-title: Experimental replication
+title: System replication (Enterprise)
 description: Dive into the documentation page for Memgraph and learn how to configure replication for data redundancy and disaster recovery. Examine the power of replication for enhanced data reliability.
 ---
 
 import { Callout } from 'nextra/components'
 
-# Experimental replication
+# System replication
+
+<Callout>
+
+Enterprise: System replication requires a Memgraph Enterprise license to
+function. For more information on Memgraph Enterprise and instructions
+on how to enable it, refer to the
+[enabling Memgraph Enterprise documentation](/database-management/enabling-memgraph-enterprise).
+
+</Callout>
```

Comment:
Is this kind of callout on other Enterprise features? If not, why it's here?

Reply:
I though we used that type of callout for enterprise features. 
Copied it from some other page.
Should we just add `(Enterprise)` to the title?

Reply:
@kgolubic I'll leave this to you, since you will know better how other Enterprise features are documented. I just noticed it's not the same for all (e.g., https://memgraph.com/docs/database-management/multi-tenancy)

Reply:
I think you are right, most of the time we just add `(Enterprise)` to the title.
I'll update